src/components/AssetIconWithBadge.tsx

```diff
@@ -101,7 +101,7 @@ const TransferIcon: React.FC<{
         showNetworkIcon={false}
         assetId={transfer.assetId}
         clipPath={getClipPath(index, totalTransfers, isNft, asset)}
-        name={transfer.asset.name}
+        name={transfer.asset.symbol}
```

Comment:
Noticed the quick glance tx row icon and transfer icon use different names which looks weird for fallback icons.